lnrpc/routerpc/config_default.go

```diff
@@ -0,0 +1,7 @@
+// +build !routerpc
+
+package routerpc
```

Comment:
Agree with @cfromknecht, this doesn' t like good imo and needs to be `routerrpc`.